src/components/DocumentationTopic/PrimaryContent/Mentions.vue

```diff
@@ -0,0 +1,56 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2024 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <section v-if="topics.length > 0" class="mentions">
+    <h2 class="mentions" id="mentions">Mentioned in</h2>
```

Comment:
Also, instead of using the raw `<h2>` here, we should probably use the `LinkableHeading` component, which will give us the behavior where you can click the heading and get a link to the subsection of the page.